ui/webui/test/README.rst

```diff
@@ -112,6 +112,18 @@ to test, you can run tests by picking a program and just executing it against th
 
     test/check-basic --machine=127.0.0.2:22000 --browser 127.0.0.2:9091
 
+Integration tests
+-----------------
+
+Integration tests run the installation to the end and do also checks on the installed system after its reboot. They are located in the `<integration>`_ subdirectory. Usually they are run by a Permian workflow provisioning rebootable virtual machines, but it is also possible to run them just by executing the test. To run them completely, including the post-reboot checks, they should be run on a en existing machine that can be rebooted as a part of the test::
+
+    WEBUI_TEST_DIR=./test test/integration/default.py --machine 192.168.122.235:22 --browser 192.168.122.235:9090
```

Comment:
Is this IP randomly chosen?

Reply:
Yes it is. I am closing this PR as this info is now included in https://github.com/rhinstaller/anaconda/pull/4667 so please follow-up there if needed.